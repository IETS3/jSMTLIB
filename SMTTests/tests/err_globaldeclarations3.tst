; checks global declarations
(set-option :global-declarations true)
(set-logic QF_UF)
(declare-fun a () Bool)
(reset)
(get-option :global-declarations) ; should be false
(assert a)  ; not defined - ERROR
