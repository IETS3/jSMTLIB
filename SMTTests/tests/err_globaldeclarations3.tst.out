success
success
success
success
false
(error "The logic must be set before an assert command is issued")
